source/docs/xrp-robot/hardware-and-imaging.rst

```diff
@@ -24,24 +24,24 @@ The XRP firmware must be downloaded and written to the Pico W. Click on ``Assets
 
 [XRP-WPILib Firmware](https://github.com/wpilibsuite/xrp-wpilib-firmware/releases)
 
+Choose the file based on the model of XRP you have. Beta versions of the XRP have 'Beta' in the name of the file. V1 releases do not have include Beta. If you choose the wrong one it will simply not be installed, just return and download the correct one.
```

Comment:
Add a link to the HW comparison that was on the firmware page so that teams can know what version of XRP they have and can then chose the right file

Reply:
Is this the link you are referring to? https://docs.sparkfun.com/SparkFun_XRP_Controller/introduction/

Reply:
@sciencewhiz ^

Reply:
yes